ArduPlane/events.cpp

```diff
@@ -166,6 +166,9 @@ void Plane::handle_battery_failsafe(const char *type_str, const int8_t action)
             // don't actually do anything, however we should still flag the system as having hit a failsafe
             // and ensure all appropriate flags are going off to the user
             break;
+        case Failsafe_Action_Parachute:
```

Comment:
Nitpick: From a consistency perspective I'd suggest maybe moving this above the Failsafe_Action_None, but it's not really critical.